src/elements/form-field/form-field/form-field.scss

```diff
@@ -388,7 +388,14 @@
 .sbb-form-field__input {
   display: flex;
 
-  :host([size='s']:is([data-input-type='input'], [data-input-type='select'])) & {
+  :host(
```

Comment:
Shouldn't the sbb-options not also receive styles for size s?

Reply:
Comparing a screenshot with the design, I see no differences in sbb-options dimensions/style
![image](https://github.com/user-attachments/assets/15f018dc-c85d-4fec-83f0-acb303b587d5)

Reply:
sorry for the confusion then, and thanks for checking it!!